node/core/approval-voting/src/lib.rs

```diff
@@ -2347,6 +2347,37 @@ async fn launch_approval<Context>(
 			},
 		};
 
+		let (ee_params_tx, ee_params_rx) = oneshot::channel();
+		sender
+			.send_message(RuntimeApiMessage::Request(
+				block_hash,
+				RuntimeApiRequest::SessionEeParamsByParentHash(
+					candidate.descriptor.relay_parent,
+					ee_params_tx,
+				),
+			))
+			.await;
+
+		let ee_params = match ee_params_rx.await {
+			Err(_) => return ApprovalState::failed(validator_index, candidate_hash),
+			Ok(Err(_)) => return ApprovalState::failed(validator_index, candidate_hash),
```

Comment:
nit: I think you can coalesce this into one line:

```suggestion
			Err(_) | Ok(Err(_)) => return ApprovalState::failed(validator_index, candidate_hash),
```